ppx/test/merlin/issue-429.t/component.re

```diff
@@ -0,0 +1,37 @@
+open React;
+open! ReasonReact;
```

Comment:
why do we need this `open`?

Reply:
i just copied the file from the original report in #429.

Reply:
I think `ReasonReact` might be going away soon. I don't think we need that module at all.

Reply:
Removed in https://github.com/reasonml/reason-react/pull/749/commits/1463abba06c7126610db86c531768b74ddb85b4a.